tracer/src/Datadog.Trace/IAST/Aspects/MongoDB/BsonAspect.cs

```diff
@@ -0,0 +1,150 @@
+// <copyright file="BsonAspect.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Linq;
+using System.Reflection;
+using Datadog.Trace.Iast.Dataflow;
+using Datadog.Trace.Iast.Helpers;
+
+#nullable enable
+
+namespace Datadog.Trace.Iast.Aspects.MongoDB;
+
+#if !NETFRAMEWORK
+/// <summary> BsonAspect class aspect </summary>
+[AspectClass("MongoDB.Bson", AspectType.Sink, VulnerabilityType.NoSqlInjection)]
+[global::System.ComponentModel.Browsable(false)]
+[global::System.ComponentModel.EditorBrowsable(global::System.ComponentModel.EditorBrowsableState.Never)]
+public class BsonAspect
+{
+    /// <summary>
+    /// MongoDB Bson aspect
+    /// </summary>
+    /// <param name="source"> source </param>
+    /// <returns> oui </returns>
+    [AspectMethodReplace("MongoDB.Bson.IO.BsonBinaryReader::ReadString()")]
+    [AspectMethodReplace("MongoDB.Bson.IO.BsonReader::ReadString()")]
+    [AspectMethodReplace("MongoDB.Bson.IO.IBsonReader::ReadString()")]
+    public static object ReadString(object source)
```

Comment:
Is this final? This looks like a test method

Reply:
oops, I just forgot to remove it